cmd/stack_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateServicesFlag(t *testing.T) {
+	t.Run("a non available service returns error", func(t *testing.T) {
+		err := validateServicesFlag([]string{"non-existing-service"})
+		require.NotNil(t, err)
```

Comment:
Nit. Use `require.Error` instead of `require.NotNil`?

Reply:
Done in https://github.com/elastic/elastic-package/pull/493/commits/18efbc9039fe57d4703260a02b91432121cdbd89